content/blog/how-to-ask-good-questions/index.md

```diff
@@ -0,0 +1,110 @@
+---
+{
+	title: "How to ask better questions",
+	description: "We all ask questions from time to time, so here are some of my favourite tips when it comes to how to improve the quality of your questions.",
+	published: '2022-07-20T20:10:03.000Z',
+	authors: ['alexchadwick'],
+	tags: ['opinion'],
+	attached: [],
+	license: 'cc-by-nc-sa-4'
+}
+---
+
+
+## Introduction
+
+In this day and age, the programming ecosystem has become so rich and complex that asking questions is inevitable for any developer, regardless of what stage of their career they find themselves in.
+
+However, it is not always easy to ask a question, and often you'll see StackOverflow questions being downvoted or taken down due to the question being of poor quality.
```

Comment:
```suggestion
However, it is not always easy to ask a question, and often you'll see StackOverflow questions being downvoted or taken down because of a sense of confusion around the question itself.
```